infra/src/main/java/com/redhat/service/bridge/infra/transformations/TransformationValidationResult.java

```diff
@@ -0,0 +1,24 @@
+package com.redhat.service.bridge.infra.transformations;
+
+public class TransformationValidationResult {
```

Comment:
@sutaakar Any reason why we can't move `ValidationResult` from the `actions` module into `infra` module and re-use it? The implementations look identical.

Reply:
good point, I haven't noticed that class, will do

Reply:
Done